glusterhealth/reports/georep.py

```diff
@@ -1,4 +1,6 @@
 from utils import process_log_file
+from gluster.cli import volume, georep
```

Comment:
@kotreshhr @aravindavk Is gluster.cli module part of gluster geo-repl ? Where can we find gluster.cli module ?

Reply:
It is part of https://github.com/gluster/glustercli-python project, I will add this as dependency for this project.